### ISSUE TITLE
[Accessibility] Some thoughts on error in line audio cue

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#175341]
CC @meganrogge

With today's insider update, no longer able to precisely identify the starting and ending point of an error via Left/RightArrows because you will hear error audio cues in the whole columns instead of the specific error columns.

1. Column-by-column (Left/RightArrow) navigation: previous behavior is ideal. Error audio cue needs to be played only on the error-causing columns instead of the whole columns.

1. Line-by-line (Up/DownArrow) navigation: current behavior is good. If there is an error in the currently focused line (even though current column focus is not at the error column position), error audio cue needs to be played.

I now understand that 1. and 2. may not be addressed altogether because VSCode does not know whether users have approached the current cursor position via "line-by-line" or "column-by-column" navigation. In any event that you need to take only one method, 1. (the previous approach) is more logical and robust.

## System Info

Version: 1.76.0-insider (system setup)
Commit: 720985f2c68cadb5e375d9ec671476c548f61769
Date: 2023-02-24T06:18:04.880Z
Electron: 19.1.10
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by meganrogge** [comment]
@jooyoungseo, we only play the audio cue when 1) the line changes and there's an error on the line or 2) when the column changes and there's an error at the current column position.  

Can you pls indicate a scenario you've encountered where the behavior is not ideal/unexpected?

**Comment by meganrogge** [comment]
related https://github.com/microsoft/vscode/issues/175348

**Comment by meganrogge** [comment]
Here's how it works currently:

- Left/right arrow navigation plays the error audio cue when within an error range
- when right arrow is used on the last column of the line and the line it jumps to contains an error
- Up /down arrow navigation plays an audio cue when it lands on an error line

**Comment by jooyoungseo** [comment]
@meganrogge 

> 2) when the column changes and there's an error at the current
column position.

As far as I tested with today's insiders, I hear audio cue outside the error range in any column position when I use Left/RightArrows as long as the line contains an error. This is unexpected.

**Comment by meganrogge** [comment]
That is unexpected and I cannot reproduce on mac. I'll try windows

**Comment by meganrogge** [comment]
works for me on windows. here's an example that works for me - where the error is just `problem`

```ts
// commenting on this line
const x = problem
```

**Comment by meganrogge** [comment]
found a way to reproduce the issue

**Comment by meganrogge** [comment]
repro:

insert `import { IConfiguration Service } from 'vs/platform/configuration/common/configuration';` and navigate via left and right arrow keys. you should only hear the error for the underlined section IE `IConfiguration Service`
